app/containers/LoginLedgerNanoS/LoginLedgerNanoS.jsx

```diff
@@ -106,6 +158,42 @@ export default class LoginLedgerNanoS extends React.Component<Props, State> {
     )
   }
 
+  async fetchInitialKeys() {
+    const publicKeys = await getPublicKeys()
+    this.setState({
+      publicKeys,
+      loadingPublicKeys: false,
+      address: {
+        value: publicKeys[0].key,
```

Comment:
would be nicer if the initial public key was a constant

```
  const initialPublicKey = publicKeys[0]
```

Reply:
Or perhaps
```
const { key: initialKey } = publicKeys[0]
```